modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -0,0 +1,297 @@
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector operation.
+|Connector Configuration |Select the global configuration you just created.
+|Message |`#[payload]`
+|Queue url |Enter the queue URL, if you did not enter a default queue URL in the global element.
+|===
++
+For example:
++
+image::amazon/amazon-sqs-send-message.png[send-message]
+
+=== Add a Logger Component to Display the Response in the Mule Console
+
+. In the Mule Palette, search for *Logger*.
+. Drag the component onto the canvas, next to the *Send Message* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the logger, such as `Log Response`.
+|Message |Sent Message: `#[payload]`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon/amazon-sqs-logger.png[logger]
+
+=== Obtain the Number of Messages in the Queue
+
+. In the Mule Palette, search for *Amazon SQS*.
+. Select the *Get approximate number of messages* operation and drag it onto the canvas, next to the *Logger* component.
++
+. Configure the following fields on the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector operation.
+|Connector Configuration |Select the global configuration you created previously for Amazon SQS Connector.
+|Message |`#[payload]`
+|Queue url |Enter the queue URL.
+|===
++
+For example:
++
+image::amazon/amazon-sqs-get-message-count.png[get-message-count]
+
+=== Add a Logger to Display the Number in the Mule Console
+
+. In the Mule Palette, search for *Logger*.
+. Drag the component onto the canvas, next to the *Get Count of Messages in queue* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the logger, such as `Log Count`.
+|Message |Approx. messages in queue: `#[payload]`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon/amazon-sqs-logger2.png[log-message-count]
+
+== Create a Flow to Receive Messages
+
+Now create another flow to receive messages and log them before deleting them from the queue.
+
+. In the Mule Palette, search for *SQS* and select the *Receivemessages* operation.
++
+image::amazon/amazon-sqs-select-receive.png[select-send-message]
++
+. Drag the *Receivemessages* operation onto the canvas.
+. Configure the following fields in the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector operation.
+|Connector Configuration |Select the global configuration you created previously for Amazon SQS Connector.
+|Number of Messages |10
+|Queue url |
+|===
++
+NOTE: If provided, the value of the *Queue URL* field takes precedence over the *Global Element Properties Queue URL* field value.
++
+For example:
++
+image::amazon/amazon-sqs-receive-message-config.png[select-send-message]
```

Comment:
Same here, the small icon will change, please update this screenshot with connector's latest version 5.4.1